NGitLab/Models/Range.cs

```diff
@@ -0,0 +1,12 @@
+using System.Text.Json.Serialization;
+
+namespace NGitLab.Models;
+
+public class Range
+{
+    [JsonPropertyName("line_code")]
+    public string LineCode { get; set; }
+
+    [JsonPropertyName("type")]
+    public string Type { get; set; }
```

Comment:
I think this too could be represented by an enum, with possible values: `old` and `new`.